stetho/src/main/java/com/facebook/stetho/common/android/FragmentCompatUtil.java

```diff
@@ -62,10 +78,12 @@ private static Object findFragmentForViewInActivity(
       FragmentCompat compat,
       Activity activity,
       View view) {
-    return findFragmentForViewInFragmentManager(
-        compat,
-        compat.forFragmentActivity().getFragmentManager(activity),
-        view);
+    Object fragmentManager = compat.forFragmentActivity().getFragmentManager(activity);
+    if (fragmentManager != null) {
```

Comment:
Is this a separate issue being fixed?  I'm fine with it in context, just curious